roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -62,17 +62,25 @@ class Interface(object):
 
         # veth interfaces named tap... belong to Neutron
         if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname) and \
-                self.iftype() == 'veth':
-            return True
+                (self.iftype() == 'veth' or self.iftype() == 'tun'):
+            if options.device is None:
+                return True
+            if "tap" in options.device:
+                return True
 
-        if re.match(r'^vxlan[\d]{1-4}$', self.ifname) and \
+        if (re.match(r'^vxlan[\d]{1-4}$', self.ifname) or re.match(r'^vxlan\-[0-9]{1,4}$', self.ifname)) and \
```

Comment:
I suspect this line is greater than 80 chars, so I'd shorten it.  Additionally, I'd consider some comments on the intent of the regex, because is mostly an alien langauge, so it's always nice to see a comment to recognize the origianl developers intent.

Additionally, pep8 on multiline if statements applies here, I'd use parenthesis and break it up for readability.
https://www.python.org/dev/peps/pep-0008/#multiline-if-statements